src/upgrades/mod.rs

```diff
@@ -1134,6 +1165,26 @@ mod tests {
         );
     }
 
+    #[test]
+    fn prepare_then_upgrade_0_14_2() {
+        test_upgrade(
+            "test-resources/migrations/v0_14_2/",
+            &[
+                "ca_objects",
+                "cas", // testbed/command-6.json was intentionally removed
+                "keys",
```

Comment:
No, not for this test. Generally speaking it may be better to leave the keys dir out actually as it contains private keys. Even if this is all test data, it's a good habit not to include them.